canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -101,6 +107,32 @@ $toolbar-divider-width: 1px;
 	}
 }
 
+.toolbar-div.toolbar-sm-div .toolbar-item {
+
+	& button {
+		padding: 0;
+		height: $toolbar-button-sm-height;
+		width: $toolbar-button-sm-width;
+		min-height: 30px;
+	}
+
+	&.default,
+	&.ghost {
+		& button {
+			height: $toolbar-button-sm-height;
+			width: $toolbar-button-sm-width;
+		}
+
+	}
+
+	&.tertiary {
+		& button {
+			height: $toolbar-button-sm-height;
+			width: $toolbar-button-sm-width;
+		}
+	}
+}
+
 .toolbar-spacer {
```

Comment:
Can you add a radio button set here to allow someone using the test harness to change between "sm" and "md"
<img width="423" alt="image" src="https://user-images.githubusercontent.com/15114439/181618741-20a087b0-f650-400a-8c7b-0293c84293de.png">

Reply:
Wrapped  `.toolbar-spacer ` within `&.toolbar-sm-div `